UI/src/app/_components/settings/settings.component.ts

```diff
@@ -161,6 +162,8 @@ export class SettingsComponent {
 	}
 
 	public navChanged(event: any) {
+		console.log(this.socketService.sourceTypes);
+		(window as any).socketService = this.socketService;
```

Comment:
why do we need to share this service using the window object?
I can't find where it is used...